### PR TITLE
Archi and kil

### PR DESCRIPTION
**Pull request by sejtalk**
## Changelog:
- Added the Warblade of Archimonde artifact.
- Added a new icon for the Scepter of Sargeras.
- Kil'jaeden also gets the Legion Doomhorn when spawned.
- Kil'jaeden now gets the Scepter of Sargeras as he owned it according to the lore.
- Added new portraits for Archimonde and Kil'jaeden.
- Archimonde is now lvl 8 warlock and lvl 1 warrior.

# How to test:
Use `event WCFEL.1000` console command, check if Archimonde has a portrait and his warblade, do the same for Kil'jaeden but check for the scepter instead.

### CODE REVIEW

File: events/wc_fel_events.txt [hunk 1]
```diff
@@ -827,7 +829,12 @@ character_event = {
 				trait = being_demon
 				trait = fair trait = proud trait = patient trait = deceitful trait = arbitrary trait = cynical trait = cruel trait = envious
 			}
-			new_character = { narrative_event = { id = WCFEL.1005 } }
+			new_character = {
+				set_character_flag = is_kiljaeden
+				add_artifact = legion_doomhorn
+				add_artifact = scepter_of_sargeras
```
Comment (zumbak04) [inline events/wc_fel_events.txt]: Not mentioned this change in the changelog.

**Comment by geffe1** [comment]
>Added Warblade of Archimonde artifact.

Eyupp, but I'd like to change the text- will try to think of something for it.
<details>
<summary>Click to expand</summary>
ck2_481
</details>
>Added new icon for Scepter of Sargeras.
Yes, but... this text makes little sense to it.
<details>
<summary>Click to expand</summary>
ck2_484
</details>
>Kilj'jaeden also gets Legion Doomhorn when spawned.
Both Kil'jaeden and Archimonde gets it, however the text for it.. could perhaps be changed a bit, it sounds weird.
<details>
<summary>Click to expand</summary>
ck2_483
</details>
>Added new portraits for Archimonde and Kil'jaeden.
Yepp, and both looks great.

**Comment by geffe1** [comment]
This humoungus warblade belongs to Archimonde the Defiler, Lord of the Burning Legion. Countless of worlds were conquered by the might of this blade and its owner.

Sounds better for Archimonde's blade.

File: events/wc_fel_events.txt (hunk 1 above)
Comment (sejtalk) [reply events/wc_fel_events.txt]: fixed

File: common/artifacts/901_wc_artifacts_weapon_twohanded.txt [hunk 2]
```diff
@@ -235,7 +235,39 @@ scepter_of_sargeras = {
...
+	flags = { combat_weapon sword fel_artifact }
+
+	active = {
+		is_adult = yes
+
+		OR = {
+			NOT = { has_class_trait_trigger = yes }
+			is_physical_class_trigger = yes
+			is_combined_class_trigger = yes
+			has_character_flag = is_archimonde
```
Comment (zumbak04) [inline common/artifacts/901_wc_artifacts_weapon_twohanded.txt]: Maybe we just could make Archimonde half-warrior half-warlock? Or not half... anyway, you got my point.

Comment (sejtalk) [reply common/artifacts/901_wc_artifacts_weapon_twohanded.txt]: well he's definitely a warlock, so i think having an exception for this item is ok

Comment (zumbak04) [reply common/artifacts/901_wc_artifacts_weapon_twohanded.txt]: If he can wield sword, he's kinda warrior. +1 warrior level won't break anything.

Comment (sejtalk) [reply common/artifacts/901_wc_artifacts_weapon_twohanded.txt]: well the blade is from rpg lore, and its only mentioned that kazzak uses archies blade after he died

Comment (zumbak04) [reply common/artifacts/901_wc_artifacts_weapon_twohanded.txt]: So, RPG lore says Archimonde is kinda warrior.
image

Comment (zumbak04) [reply common/artifacts/901_wc_artifacts_weapon_twohanded.txt]: Also, I can use the blade, but the tooltip can't say why.
<details>
<summary>Click to expand</summary>

image

</details>

File: interface/portrait_properties/00_portrait_properties.txt [hunk 3]
```diff
@@ -4752,4 +4752,34 @@
...
+				has_portrait_property = { layer = 27 index = 2 }
+			}
+		}
+	}
+	1 = {		# Archimonde
+		factor = 100
+		trigger = {
+			has_game_rule = { name = static_portraits value = enabled }
+			
+			has_character_flag = is_archimonde
```
Comment (zumbak04) [inline interface/portrait_properties/00_portrait_properties.txt]: Dead characters lose flags, so portrait may disappear. Suggest using hidden traits instead.
https://ck2.paradoxwikis.com/trait%20modding

<details>
<summary>Click to expand</summary>

image
image

</details>

Comment (sejtalk) [reply interface/portrait_properties/00_portrait_properties.txt]: fixed

**Pull request merged**